client/src/main/scripts/components/search/search-index.json

```diff
@@ -1 +1 @@
-{"hash":"0e3b208baaaf58a032a24251a34748634d46ecb6"}
\ No newline at end of file
+{"hash":"b0ae8fc301e6a47119bd28ed91f66552043516a1"}
```

Comment:
**3)** This is a hash of the search-index-generation-process. It changes when the facts change, and also when the index configuration changes.  It is generated automatically by `:lucene:indexGenerate`.